docs/1.Welcome/What is Checkov.md

```diff
@@ -35,8 +35,7 @@ Checkov integrates with advanced features in the [Bridgecew platform](https://br
 
 Bridgecrew can validate the same Checkov IaC policies against your runtime cloud environments in AWS, Azure and Google Cloud, allowing you to find and fix issues in existing deployments and detect cloud drifts. Read more in [Bridgecrew's documentation](https://docs.bridgecrew.io/docs/step-2-integrate-with-cloud-provider).
 
-
-### Pull Request Annotations 
+### Pull Request Annotations
```

Comment:
why is this change required?

Reply:
Again, a change made by my code formatter. I've reverted this back to its original state